src/config/properties.rs

```diff
@@ -184,12 +184,13 @@ pattern = 'prop\d+'
 "#,
         );
         let props = Properties::parse(Some(&toml), "a");
-        assert_eq!(props.len(), 4);
```

Comment:
Why did this change?

Reply:
The explanation is in the comment just below. ;)

Reply:
Which comment?